code/index.html

```diff
@@ -24,31 +24,31 @@
         <div class="icon-office">
         </div>
         <ul>
-          <li><a href="file:///Users/BigBen/CodeFellows/301/portfolio/code/index.html">Home</a></li>
-          <li><a href="">About</a></li>
+          <li class="tab" data-content="projects"><a href="file:///Users/BigBen/CodeFellows/301/portfolio/code/index.html">Home</a></li>
+          <li class="tab" data-content="about"><a href="">About</a></li>
           <li><a href="https://www.linkedin.com/in/william-shields-87740582?trk=nav_responsive_tab_profile_pic" target="_blank"> LinkedIn</a></li>
           <li><a href="https://github.com/iamrobinhood12345" target="_blank"> GitHub</a></li>
           <li><a href="https://shieldsbooks.wordpress.com/2016/06/06/25/" target="_blank"> Shields Books</a></li>
         </ul>
       </nav>
     </header>
     <main>
-      <section id="projects">
+      <section id="projects" class="tab-content">
         <article class="template">
           <header>
             <h1><a href="">
               Title
             </a></h1>
-            <div class="publishedLine">
+            <p class="publishedLine">
               <!--Published <time pubdate datetime="2000-01-01">Publish Time</time>-->
               Published on
-            </div>
+            </p>
             <!--<section class="article-body"><p>Working on this website. Under construction!</p></section>-->
           </header>
           <section class="article-body"></section>
         </article>
       </section>
-      <section id="about">
+      <section id="about" class="tab-content">
```

Comment:
Suggestion to add a style for this section so that it is initially hidden on the home page.